sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -344,6 +344,20 @@ object functions {
    */
   def abs(e: Column): Column = Abs(e.expr)
 
+  /**
+   * Aggregate func:compute the stddev of the specific column.
+   *
+   * @group agg_funcs
+   */
+  def stddev(columnName: String): Column = stddev(Column(columnName))
+
+  /**
+   * Aggregate func:compute the stddev of the specific column
+   *
+   * @group agg_funcs
+   */
+  def stddev(e: Column): Column = StdDeviation(e.expr)
```

Comment:
can you move these two functions to the aggregate function section, and change the comment to say

```
Aggregate function: Computes the standard deviation of the specific column.
```